### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by cornerot**
Sensus communis

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -156,14 +156,12 @@ Signed,
...
 
 [unti1x](https://github.com/unti1x)
 
 [Daniel](https://github.com/Zuzuk-null)
 
 [Sherzod Mutalov](https://github.com/shmutalov)
 
-[Dmitry Derbin](https://github.com/lmcdd)
-
-[N0rbert](https://github.com/N0rbert)
```
Comment (nekonee) [inline index.md:169]: Why are you deleting those lines?

File: index.md [hunk 2]
```diff
@@ -156,14 +156,12 @@ Signed,
 
 [Adam Katz](https://github.com/dial8gue)
 
-[Watcher3056] (https://github.com/Watcher3056)
```
Comment (nekonee) [inline index.md:159]: Why are you deleting this line?

**Comment by podvornyakva** [comment]
Please. add a file instead of edit index.md

Follow README.md instructions.

Thank you.

**Comment by nukeop** [comment]
Please see readme for the format your signature needs to follow. Once you fix the problem, please reopen the PR.

**Pull request closed without merging**